### PR TITLE
Add tooltips to "kubelet" term definition

### PR DESCRIPTION
**Pull request by kirisakow**
attempt to unblock #42046

### CODE REVIEW

**Comment by kirisakow** [comment]
@sftim 
@onlydole

File: content/en/docs/reference/glossary/cri.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ tags:
 - fundamental
 ---
 The container runtime interface (CRI) is an API for container runtimes
-to integrate with kubelet on a node.
+to integrate with {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} on a node.
```
Comment (shannonxtreme) [inline content/en/docs/reference/glossary/cri.md:15]:
```suggestion
to integrate with the {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} on a node.
```

File: content/en/docs/reference/glossary/cri.md
Comment (shannonxtreme) [inline content/en/docs/reference/glossary/cri.md:1]: Not for this PR, but why do we have separate files for CRI and Container Runtime Interface? They're the same thing, we should delete the duplicate and merge the content if needed.

**Review by shannonxtreme — COMMENTED** [review]
LGTM

**Comment by sftim** [comment]
/retitle Add tooltips to "kubelet" term definition

Please avoid mentioning issue IDs in PR descriptions.

**Pull request closed without merging**